### PR TITLE
org.squey.Squey

### PR DESCRIPTION
**Pull request by jbleonesio**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
  **--socket=x11** is required to show windows using X11
  **--share=network** is required so the app can connect to remote servers.
  **--share=ipc** is required to share host namespace IPC
  **--filesystem=host** is required to access very large datasets that are stored outside of flatpak on fast devices like RAID-0 arrays of SSDs
  **--device=dri** is required to access GPUs
  **--allow=devel** is required to generate crash reports
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://gitlab.com/squey/squey
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

**Note: this is a package rename of [com.gitlab.inendi.Inspector](https://github.com/flathub/com.gitlab.inendi.Inspector/)**

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by jbleonesio** [comment]
bot, build org.squey.Squey

File: org.squey.Squey.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - --share=network
+  - --share=ipc
+  - --filesystem=host
+  - --filesystem=/opt/squey
+  - --filesystem=~/.squey
+  - --filesystem=/tmp
+  - --device=dri
+  - --allow=devel
+modules:
+  - name: inendi-inspector
```
Comment (hfiguiere) [inline org.squey.Squey.yaml:17]: this name doesn't really match.

File: flathub.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: this is disappointing.

Comment (jbleonesio) [reply flathub.json:2]: See https://github.com/flathub/flathub/pull/3272#discussion_r897950584

File: org.squey.Squey.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+sdk: org.freedesktop.Sdk
+command: squey_launcher.sh
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --share=ipc
+  - --filesystem=host
+  - --filesystem=/opt/squey
+  - --filesystem=~/.squey
+  - --filesystem=/tmp
```
[the remark this answers is not in the document]
Comment (jbleonesio) [reply org.squey.Squey.yaml]: I fixed the redundancy, for the broad filesystem permissions see : https://github.com/flathub/flathub/pull/3272#discussion_r897948743

**Comment by jbleonesio** [comment]
As stated in the PR description, please note that this package is a rename of [com.gitlab.inendi.Inspector](https://github.com/flathub/com.gitlab.inendi.Inspector/)

**Comment by jbleonesio** [comment]
bot, build org.squey.Squey

**Comment by barthalion** [comment]
bot, build org.squey.Squey

**Pull request closed without merging**